src/System.Binary/System/Binary/BufferReader_BE.cs

```diff
@@ -22,22 +22,276 @@ public static partial class Binary
         public static T ReadBigEndian<[Primitive]T>(this Span<byte> buffer) where T : struct
             => BitConverter.IsLittleEndian ? UnsafeUtilities.Reverse(buffer.Read<T>()) : buffer.Read<T>();
 
+        #region ReadBigEndianROSpan
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool TryReadBigEndiann<[Primitive]T>(this ReadOnlySpan<byte> buffer, out T value) where T : struct
+        public static short ReadBigEndianInt16(this ReadOnlySpan<byte> buffer)
```

Comment:
I think the name should be ReadInt16BigEndian as opposed to ReadBigEndian16. The former is better for intellisense completion, as you have to type less before you are down to one matching API.